llama.cpp

```diff
@@ -4560,7 +4560,7 @@ static void llm_load_vocab(
                 [](unsigned char c){ return std::tolower(c); });
 
             if (gen_name.find("code") != std::string::npos) {
-                if (model.arch == LLM_ARCH_LLAMA) {
+                if (model.arch == LLM_ARCH_LLAMA && gen_name.find("llama") != std::string::npos) {
```

Comment:
This is not a good idea, we need something better

Reply:
Ok, turns out that was a little simpler to do after moving the code-block down a little.